src/helpers/highlight.ts

```diff
@@ -4,7 +4,8 @@ import { TAG_REPLACEMENT } from '../lib/escape-highlight';
 import { component } from '../lib/suit';
 
 export type HighlightOptions = {
-  attribute: string;
+  // @MAJOR only accept array of paths here
```

Comment:
`string` and `string[]` probably. You don't want to force an array since a string covers 95% use cases. Besides, it'll mean renaming the option to `attributePath`.

Reply:
so you're suggesting no change in a future major then? if we'd move to the array form only in the docs, doesn't it also make sense to in time move to the array form only here?

Reply:
I meant the `string` type would be for attributes like `label`, and wouldn't support nested attributes (e.g., `lvl0.label`). We'd dev warn about it and guide toward the `string[]` type.

Reply:
I see, good point!

Reply:
Updated in 19514d4